internal/handler/name_handler.go

```
@@ -0,0 +1,124 @@
+}
+
+// CreateName은 새로운 이름을 생성하는 HTTP 엔드포인트 핸들러입니다.
+// POST 요청의 body에서 JSON 형식의 이름을 받아 저장합니다.
+func (h *NameHandler) CreateName(w http.ResponseWriter, r *http.Request) {
+    var req NameRequest
+
+    // null check validation
+    err := json.NewDecoder(r.Body).Decode(&req)
+    if err != nil || req.Name == "" {
+        w.WriteHeader(http.StatusBadRequest)
+        json.NewEncoder(w).Encode(ErrorResponse{
```

golangci-lint (1.64.8)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)